lib/Test/Nginx/Socket.pm

```diff
@@ -3662,7 +3662,7 @@ Below is an example from ngx_headers_more module's test suite:
     --- response_headers
     ! X-Foo
     --- response_body
-    x-foo: 
+    x-foo:
```

Comment:
Don't do this. This line trailing space is intentional and required.

Reply:
Added back.